pubspec.yaml

```diff
@@ -6,47 +6,50 @@ environment:
   sdk: ">=2.19.2 <3.0.0"
 
 dependencies:
-  # objectbox_flutter_libs: ^1.7.2
-  firebase_core: ^2.8.0
-  screen_state: ^2.0.0
+  app_minimizer: ^1.0.0+2
+  async: ^2.10.0
+  audioplayers: ^5.2.0
+  cloud_firestore: ^4.4.5
   cupertino_icons: ^1.0.2
-  get: 4.6.5
+  file_picker: ^5.5.0
+  firebase_core: ^2.8.0
+  fl_location: ^2.0.0+1
+  flutter:
+    sdk: flutter
+  flutter_expandable_fab: ^1.8.1
+  flutter_fgbg: ^0.2.2
   flutter_foreground_task: ^6.0.0
+  flutter_launcher_icons: ^0.13.1
+  flutter_map: ^4.0.0
   flutter_ringtone_player: ^3.2.0
+  flutter_secure_storage: ^8.0.0
+  flutter_svg: ^2.0.4
   flutter_time_picker_spinner: ^2.0.0
-  cloud_firestore: ^4.4.5
+  get: 4.6.5
+  google_sign_in: ^6.0.2
+  hive: ^2.2.3
+  hive_flutter: ^1.1.0
```

Comment:
Why another database? We already have IsarDB and Secure Storage which can be used to store the path...

Reply:
I could have missed it will implement that in next after implying the suggested changes